webapp/sources/rudder/rudder-web/src/main/javascript/rudder/rudder-datatable.js

```diff
@@ -1365,6 +1366,9 @@ function createNodeTable(gridId, refresh) {
       bottom = $('.main-details > .table-container'),
       handle = $('#drag');
 
+    if (handle) {
```

Comment:
what is the type handle ? if empty is undefined you should do "handle !== undefined"

Reply:
dynamic typing of js could lead to true/false when you don't want

Reply:
it's a jQuery selector result and the check should be `handle.length` as mentionned here : https://stackoverflow.com/a/299821

Reply:
I added a `> 0` for the sake of it :smile: